### PR TITLE
Py plumed

### PR DESCRIPTION
**Pull request by GiovanniBussi**
Reopened #8420 after some cleanup.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by GiovanniBussi** [comment]
In my previous PR (#8420) I was planning to merge this recipe and the plumed recipe using `outputs:`. However, this looked more complicated than I expected. Moreover, these two recipes do not need to be strictly in sync. I thus would prefer to add a separate recipe for py-plumed as originally suggested by @chrisburr.

The recipe seems to work correctly. However, in the [Linux build](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=37842&view=logs) I see an error like this one:
````
ERROR :: get_rpaths_raw()=['/home/conda/feedstock_root/build_artifacts/py-plumed_1558946620167/_build_env/lib:/home/conda/feedstock_root/build_artifacts/py-plumed_1558946620167/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib'] and patchelf=/home/conda/feedstock_root/build_artifacts/py-plumed_1558946620167/_build_env/lib:/home/conda/feedstock_root/build_artifacts/py-plumed_1558946620167/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib disagree for /home/conda/feedstock_root/build_artifacts/py-plumed_1558946620167/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.6/site-packages/plumed.cpython-36m-x86_64-linux-gnu.so ::
````

I don't know if it indicates a problem or not. If this is ok, I would be happy if this PR gets merged. Thanks!

File: recipes/py-plumed/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (chrisburr) [inline recipes/py-plumed/meta.yaml]: It looks like you're using `make`?
```suggestion
    - {{ compiler('cxx') }}
    - make
```

File: recipes/py-plumed/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd python
+make pip
```
Comment (chrisburr) [inline recipes/py-plumed/build.sh]: Can we use multiple jobs here?
```suggestion
make -j${CPU_COUNT} pip
```

Comment (GiovanniBussi) [reply recipes/py-plumed/build.sh]: This target is just copying two files from another directory, so I think it does not make sense to parallelize it:
````
pip:
        cp ../src/wrapper/Plumed.h .
        cp ../VERSION .
````

**Pull request merged**